test/common/CollectiveArgs.hpp

```diff
@@ -72,6 +74,20 @@ namespace RcclUnitTesting
     char ptr[MAX_RANKS * sizeof(double)];
   };
 
+  struct OptionalColArgs
+  {
+    ncclRedOp_t redOp = ncclSum;
+    int         root = 0; // Used as "peer" for Send/Recv
+    PtrUnion    localScalar = {nullptr};
+    int         scalarMode = -1; // -1 if scalar not used
+    // allToAllv args
+    size_t      sendcounts[MAX_RANKS*MAX_RANKS];
+    size_t      sdispls[MAX_RANKS*MAX_RANKS];
+    size_t      recvcounts[MAX_RANKS*MAX_RANKS];
+    size_t      rdispls[MAX_RANKS*MAX_RANKS];
```

Comment:
Still think these should be renamed to make it more obvious to newcomers.
numElemToSend
sendElemOffset
numElemToRecv
recvElemOffset